.github/workflows/lib-injection-test.yml

```diff
@@ -31,20 +23,58 @@ jobs:
     env:
       TEST_LIBRARY: dotnet
       WEBLOG_VARIANT: 'dd-lib-dotnet-init-test-app'
-      LIBRARY_INJECTION_CONNECTION: ${{ inputs.lib_injection_connection }}
-      LIBRARY_INJECTION_ADMISSION_CONTROLLER: ${{ inputs.lib_injection_use_admission_controller == true && 'use-admission-controller' || '' }}
       DOCKER_REGISTRY_IMAGES_PATH: ghcr.io/datadog
       DOCKER_IMAGE_TAG: ${{ inputs.commit_id }}${{ inputs.runtime == 'alpine' && '-musl' || '' }}
       DOCKER_IMAGE_WEBLOG_TAG: ${{ inputs.commit_id }}-${{ inputs.runtime }}
       RUNTIME: ${{ inputs.runtime }}
       BUILDX_PLATFORMS: linux/amd64
-      MODE: manual
     steps:
-      - name: lib-injection test runner
-        id: lib-injection-test-runner
-        uses: DataDog/system-tests/lib-injection/runner@main
```

Comment:
I wonder, is there any way to encapsulate all this in a github action like was being done before? 🤔 Seems like a maintenance headache to have to go through and update this across all repos if something changes?

Reply:
> I wonder, is there any way to encapsulate all this in a github action like was being done before? 🤔 Seems like a maintenance headache to have to go through and update this across all repos if something changes?

You are right. Same case for system-tests and parametric. I have decided to discard this option for the moment, since the composite we were using for lib-injection, had some references to the main branch and this also caused us problems.
But yes, in the future it should be the way to go, to build common and reusable actions from all repositories (dd-trace-XYZ)...